lib/driver.js

```diff
@@ -190,6 +198,19 @@ class XCUITestDriver extends BaseDriver {
       if (_.has(this.opts, 'useJSONSource')) {
         await this.updateSettings({useJSONSource: this.opts.useJSONSource});
       }
+
+      let wdaSettings = {
+        elementResponseAttributes: DEFAULT_SETTINGS.elementResponseAttributes,
+        shouldUseCompactResponses: DEFAULT_SETTINGS.shouldUseCompactResponses,
+      };
+      if (_.has(this.opts, 'elementResponseAttributes')) {
+        wdaSettings.elementResponseAttributes = this.opts.elementResponseAttributes;
```

Comment:
`this.opts.elementResponseAttributes` and `this.opts.shouldUseCompactResponses`. Are these desired caps?

Reply:
yes, caps that are used to set the initial settings values